test/install.js

```
@@ -0,0 +1,43 @@
+const fs = require('fs')
+const path = require('path')
+const test = require('tape')
+const execa = require('execa')
+const rimraf = require('rimraf')
+
+/*
+  Test that go-ipfs is downloaded during npm install.
+  - package up the current source code with `npm pack`
+  - install the tarball into the example project
+  - ensure that the "go-ipfs.version" prop in the package.json is used
+*/
+
+const testVersion = 'v0.4.21-rc3'
+let tarballName = null
+
+function packTarball () {
+  try {
+    const parentDir = path.join(__dirname, '..')
+    const res = execa.sync('npm', ['pack', parentDir], {
+      cwd: __dirname
+    })
+    tarballName = res.stdout
+    return tarballName
+  } catch (err) {
+    console.error(err)
+  }
```

Let this throw? The test won't pass if this fails right?